spec/queries/keywords_query_spec.rb

```diff
@@ -4,36 +4,177 @@
 
 RSpec.describe KeywordsQuery, type: :query do
   describe '#call' do
-    context 'given the user search for ads_top_urls' do
-      context 'given ads_top_urls contain the filtering word vpn' do
-        it 'returns 2 keywords' do
+    context 'when querying with adwords_url_contains' do
```

Comment:
should we make to wording consistent with this

'given querying with adwords_url_contains`